config/cable.yml

```diff
@@ -2,9 +2,9 @@ development:
   adapter: async
 
 test:
-  adapter: async
+  adapter: test
 
 production:
   adapter: redis
-  url: redis://localhost:6379/1
+  url: <%= ENV.fetch("REDIS_URL") { "redis://localhost:6379/1" } %>
```

Comment:
Do we need to do anything differently for the deploy with this updated env variable?
`REDIS_URL`